emuu-client/src/components/CreatorsPage/CreatorsPage.js

```diff
@@ -0,0 +1,79 @@
+import "./UserInfo.scss";
+import "./Feeds.scss";
+import { Avatar } from "@mui/material";
+import { AxiosContext } from "react-axios/lib/components/AxiosProvider";
+import React, { useState, useEffect } from "react";
+import AddIcon from "@mui/icons-material/Add";
+import "./Profile.scss";
+import "../../Firebase.js";
+import Feeds from "./Feeds";
+import UserInfo from "./UserInfo";
+import { ref, getStorage, uploadBytes, getDownloadURL } from "firebase/storage";
+import { db, storage } from "../../Firebase.js";
+import {
+  getDoc,
+  getDocs,
+  setDoc,
+  doc,
+  collection,
+  query,
+  where,
+  onSnapshot,
+  increment,
+  updateDoc,
+} from "firebase/firestore";
+
+//Function to display creator page
+function Creator({ video, setUserProfile }) {
+  const [creatorName, setCreatorName] = useState("Temp");
+  const docRef = doc(db, "Users", creatorName);
+
+  useEffect(async () => {
+    if (video) {
+      localStorage.setItem("video", JSON.stringify(video));
+    }
+    if (localStorage.getItem("video")) {
+      let video = JSON.parse(localStorage.getItem("video"));
+      setCreatorName(video.Username);
+    }
+    if (!video && !localStorage.getItem("video")) {
```

Comment:
Don't think this is necessary for the creator's page, why would we redirect to home page if there's no video?